crates/gosub_pipeline/viewpage.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+. tools/souper/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: tools/souper/bin/activate was not specified as input (see shellcheck -x).

(SC1091)